single_include/nlohmann/json.hpp

```diff
@@ -4344,9 +4344,9 @@ class optional : public std::optional<T>
         return *this;
     }
 
-    constexpr optional() noexcept = default;
+    constexpr optional() noexcept(noexcept(std::optional<T>())) = default;
 
-    constexpr optional(std::nullopt_t) noexcept
+    constexpr optional(std::nullopt_t _) noexcept
```

Comment:
Try `optional(std::nullopt_t /*unused*/)` if clang-tidy, etc. complain.

Reply:
My fix doesn't work. Eventually, I replaced the unused names with comments.